File: app/src/main/java/com/sagar/watchnext/network/models/people/moviecredits/Crew.java

package com.sagar.watchnext.network.models.people.moviecredits;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Crew {

    @SerializedName("id")
    private int id;

    @SerializedName("department")
    private String department;

    @SerializedName("original_language")
    private String originalLanguage;

    @SerializedName("original_title")
    private String originalTitle;

    @SerializedName("job")
    private String job;

    @SerializedName("overview")
    private String overview;

    @SerializedName("vote_count")
    private float voteCount;

    @SerializedName("video")
    private boolean video;

    @SerializedName("poster_path")
    private String posterPath;

    @SerializedName("backdrop_path")
    private String backdropPath;

    @SerializedName("title")
    private String title;

    @SerializedName("popularity")
    private float popularity;

    @SerializedName("genre_ids")
    private List<Integer> genreIds = null;

    @SerializedName("vote_average")
    private float voteAverage;

    @SerializedName("adult")
    private boolean adult;

    @SerializedName("release_date")
    private String releaseDate;

    @SerializedName("credit_id")
    private String creditId;

    public int getId() {
        return id;
    }

    public String getDepartment() {
        return department;
    }

    public String getOriginalLanguage() {
        return originalLanguage;
    }

    public String getOriginalTitle() {
        return originalTitle;
    }

    public String getJob() {
        return job;
    }

    public String getOverview() {
        return overview;
    }

    public float getVoteCount() {
        return voteCount;
    }

    public boolean isVideo() {
        return video;
    }

    public String getPosterPath() {
        return posterPath;
    }

    public String getBackdropPath() {
        return backdropPath;
    }

    public String getTitle() {
        return title;
    }

    public float getPopularity() {
        return popularity;
    }

    public List<Integer> getGenreIds() {
        return genreIds;
    }

    public float getVoteAverage() {
        return voteAverage;
    }

    public boolean isAdult() {
        return adult;
    }

    public String getReleaseDate() {
        return releaseDate;
    }

    public String getCreditId() {
        return creditId;
    }
}

File: app/src/main/java/com/sagar/watchnext/network/models/people/moviecredits/Crew.kt
package com.sagar.watchnext.network.models.people.moviecredits

import com.google.gson.annotations.SerializedName

class Crew {

    @SerializedName("id")
    val id: Int = 0

    @SerializedName("department")
    val department: String? = null

    @SerializedName("original_language")
    val originalLanguage: String? = null

    @SerializedName("original_title")
    val originalTitle: String? = null

    @SerializedName("job")
    val job: String? = null

    @SerializedName("overview")
    val overview: String? = null

    @SerializedName("vote_count")
    val voteCount: Float = 0.toFloat()

    @SerializedName("video")
    val isVideo: Boolean = false

    @SerializedName("poster_path")
    val posterPath: String? = null

    @SerializedName("backdrop_path")
    val backdropPath: String? = null

    @SerializedName("title")
    val title: String? = null

    @SerializedName("popularity")
    val popularity: Float = 0.toFloat()

    @SerializedName("genre_ids")
    val genreIds: List<Int>? = null

    @SerializedName("vote_average")
    val voteAverage: Float = 0.toFloat()

    @SerializedName("adult")
    val isAdult: Boolean = false

    @SerializedName("release_date")
    val releaseDate: String? = null

    @SerializedName("credit_id")
    val creditId: String? = null
}
